cypress-tests/cypress/e2e/PaymentUtils/Cybersource.js

```diff
@@ -366,5 +373,98 @@ export const connectorDetails = {
         },
       },
     },
+    PaymentMethodIdMandateNo3DSAutoCapture: {
+      Request: {
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "USD",
+        mandate_data: null,
+        customer_acceptance: {
+          acceptance_type: "offline",
+          accepted_at: "1963-05-03T04:07:52.723Z",
+          online: {
+            ip_address: "125.0.0.1",
+            user_agent: "amet irure esse",
+          },
```

Comment:
we can take this up later

Reply:
Since these changes are introduced in this PR and still need more review, I thought of getting this right in this PR itself. If other reviewers are okay with this, I've no issues.

Reply:
This not introduced in this PR , this is replicated from mandate_id test cases , so we can have other PR for all the Tests

Reply:
Cool, we should take this up in the coming PR and not leave it as it is.

Reply:
sure